src/components/QueueCardList.js

```diff
@@ -81,8 +83,8 @@ const QueueCardList = props => {
       <Col>
         <Card className="bg-light">
           <CardBody className="text-center">
-            There aren&apos;t any {props.openQueue ? 'open' : 'closed'} queues
-            right now
+            There aren&apos;t any {props.openQueue ? 'open' : 'closed'}{' '}
+            {props.isStarred ? 'starred' : ''} queues right now
```

Comment:
Just an overall comment about `QueueCardList`:
I feel slightly uncomfortable with how tightly everything is coupled together and the slight increase in complexity. `QueueCardList` is no longer just a simple component that takes in a list of queues and displays them, but it also now takes in two extra props just to alter the text that's shown when there are no queues to display.

To make things easier, we could change our `/queues` endpoint so that each queue has a `starred` attribute to indicate whether it's starred by the current user. This has the additional advantage of reducing the number of state variables we need to keep track of, and we get cross-tab star syncing for free. For modifying the text that's shown when there are no queues, I think it would be easiest to just pass it into the component as a string.

Thoughts?

Reply:
One problem with this is that it breaks the way that live updating queues via the websocket works. Currently, all users receive the same "updated" message. Having a `starred` attribute means that we'd need to send a separate message to every user.

Reply:
That's true, although the current method doesn't have socket syncing either for the same reason.